add update run docs (#54)
kubefleet-dev/website: content/en/docs/troubleshooting/ClusterStagedUpdateRun.md

> ---

Reviewer: The title of this file should be StagedUpdate.md

> The condition clearly indicates the initialization failed. And the condition message gives more details about the failure. 

Reviewer 2: ```suggestion
The condition clearly indicates the initialization failed with a message to give more details about the failure. 
```

> In this case, I used a not-existing resource snapshot index `1` for the updateRun.

Reviewer 2: ```suggestion
In this case, a non-existing resource snapshot index (`1` ) was used for the updateRun.
```
